@stellar/freighter-api/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stellar/freighter-api",
-  "version": "1.0.1",
+  "version": "1.2.0",
```

Comment:
Why are we going to 1.2.0? Shouldn't it be 1.1.0?

Reply:
This change looks like it is adding new capabilities, not fixing existing capabilities, so I'd vote it is a minor version bump, i.e. 1.2.0.

Reply:
Bah, you're right. Should be `1.1.0`